server/controllers/allocationsController.ts

```diff
@@ -246,13 +255,18 @@ export default class AllocationsController {
         'required.isSensitive': "Select 'Yes' if this includes sensitive information",
       }
     )
-
+    await this.cacheService.saveDecisionEvidence(
```

Comment:
I think this needs to be after the error checks... unless we catch all errors and notify the user of a redis error?

Reply:
I put it before the error check for partially submitted forms, if they submit a partially submitted form, log off and log back on I would have thought they would still want the fields they filled out even though it errors so that they can continue where they left off?

Reply:
Yeah but we know the text they submitted without saving it don't we, so why save it?

Reply:
we wouldn't know if they logged out if we store it in session so they wouldn't be able to continue filling out the form. 

If we save it on any submission they will always be able to continue filling it out at a later date